docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,2555 @@
+
+.. _audio-command-local-search:
+
+""""""""""""
+local search
+""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]local search <search_words>
+
+**Description**
+
+Search for songs across all localtracks folders.
+
+.. _audio-command-now:
+
+^^^
+now
+^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]now 
+
+**Description**
+
+Now playing.
+
+.. _audio-command-pause:
+
+^^^^^
+pause
+^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]pause 
+
+**Description**
+
+Pause or resume a playing track.
+
+.. _audio-command-percent:
+
+^^^^^^^
+percent
+^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]percent 
+
+**Description**
+
+Queue percentage.
+
+.. _audio-command-play:
+
+^^^^
+play
+^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]play <query>
+
+**Description**
+
+Play a URL or search for a track.
+
+.. _audio-command-playlist:
+
+^^^^^^^^
+playlist
+^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]playlist 
+
+**Description**
+
+Playlist configuration options.
+
+Scope info:
+​ ​ ​ ​ **Global**:
+​ ​ ​ ​ ​ ​ ​ ​ Visible to all users of this bot.
+​ ​ ​ ​ ​ ​ ​ ​ Only editable by bot owner.
+​ ​ ​ ​ **Guild**:
+​ ​ ​ ​ ​ ​ ​ ​ Visible to all users in this guild.
+​ ​ ​ ​ ​ ​ ​ ​ Editable by bot owner, guild owner, guild admins, guild mods, DJ role and playlist creator.
+​ ​ ​ ​ **User**:
+​ ​ ​ ​ ​ ​ ​ ​ Visible to all bot users, if --author is passed.
+​ ​ ​ ​ ​ ​ ​ ​ Editable by bot owner and creator.
+
+.. _audio-command-playlist-append:
+
+"""""""""""""""
+playlist append
+"""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]playlist append <playlist_name_OR_id> <track_name_OR_url> [args]
+
+**Description**
+
+Add a track URL, playlist link, or quick search to a playlist.
+
+The track(s) will be appended to the end of the playlist.
+
+**Usage**:
+​ ​ ​ ​ ``[p]playlist append playlist_name_OR_id track_name_OR_url [args]``
+
+**Args**:
+​ ​ ​ ​ The following are all optional:
+​ ​ ​ ​ ​ ​ ​ ​ --scope <scope>
+​ ​ ​ ​ ​ ​ ​ ​ --author [user]
+​ ​ ​ ​ ​ ​ ​ ​ --guild [guild] **Only the bot owner can use this**
+
+**Scope** is one of the following:
+​ ​ ​ ​ Global
+​ ​ ​ ​ Guild
+​ ​ ​ ​ User
+
+**Author** can be one of the following:
+​ ​ ​ ​ User ID
+​ ​ ​ ​ User Mention
+​ ​ ​ ​ User Name#123
+
+**Guild** can be one of the following:
+​ ​ ​ ​ Guild ID
+​ ​ ​ ​ Exact guild name
+
+Example use:
+​ ​ ​ ​ ``[p]playlist append MyGuildPlaylist Hello by Adele``
+​ ​ ​ ​ ``[p]playlist append MyGlobalPlaylist Hello by Adele --scope Global``
+​ ​ ​ ​ ``[p]playlist append MyGlobalPlaylist Hello by Adele --scope Global --Author Draper#6666``
+
+.. _audio-command-playlist-copy:
+
+"""""""""""""
+playlist copy
+"""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]playlist copy <id_or_name> [args]
+
+**Description**
+
+Copy a playlist from one scope to another.
+
+**Usage**:
+​ ​ ​ ​ ``[p]playlist copy playlist_name_OR_id [args]``
+
+**Args**:
+​ ​ ​ ​ The following are all optional:
```

Comment:
This arguments section currently looks a little odd with the placement of all the args.
<img width="751" alt="Screen Shot 2021-04-06 at 4 06 30 PM" src="https://user-images.githubusercontent.com/20862007/113788575-22800780-96f2-11eb-9935-a91fd1830bbe.png">